lib/scrabble.rb

```diff
@@ -1,9 +1,26 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize(word)
+    @word = word
+
+  end
 
   def score(word)
-    1
+    letters = word.upcase.split('h')
+
+    total = 0
```

Comment:
When you have a local variable outside of an enumerable and the enumerable is modifying it, it is a sign that there is a better purpose built enumerable for it.